library/kotlin/io/envoyproxy/envoymobile/filters/RequestFilterCallbacks.kt

```diff
@@ -22,4 +22,9 @@ interface RequestFilterCallbacks {
    * on the stream.
    */
   fun resetIdleTimer()
+
+
+  fun pauseRequestTimeout()
```

Comment:
It seems worth calling out that there are many timeouts that this will not pause, and that this would only affect the idle timer/timeout.